src/site/content/en/blog/layout-instability-api/index.md

```diff
@@ -236,9 +236,40 @@ const observer = new PerformanceObserver((list) => {
   }
 })
 
-observer.observe({entryTypes: ['layoutShift']});
+observer.observe({type: 'layout-shift', buffered: true});
 ```
 
+<div class="w-aside w-aside--note">
+
+**Note:** The
+[`buffered`](https://w3c.github.io/performance-timeline/#dom-performanceobserverinit-buffered)
+flag in the above example (supported in Chrome 77+) gives you access to entries
+that may have occurred prior to creating the performance observer.
+
+</div>
+
+<div class="w-aside w-aside--warning">
+
+**Beware:** The `entryType` value for this API has changed a few times during
+the experimentation period. In Chrome 76 it was `layoutShift` and in Chrome
+74-75 it was `layoutJank`.
+
+Developers implementing the stable API should only need to observe the current
+`layout-shift` value (as shown in the example above), but developers who are
+part of the origin trial may need to observe multiple entry types to cover
+their full user base).
+
+```js
+if (PerformanceObserver.supportedEntryTypes &&
+    PerformanceObserver.supportedEntryTypes.includes('layout-shift')) {
+  observer.observe({type: 'layout-shift', buffered: true});
+} else {
+  observer.observe({entryTypes: ['layoutShift', 'layoutJank']});
```

Comment:
Should we include the use of getEntriesByType for the version that doesn't have the buffered flag?

Reply:
Hmmm, that would make the example quite a bit longer (since we'd have to include the original callback). What about just linking to your example (assuming that's not going to be deleted)?

Reply:
That sounds fine.